mappings/net/minecraft/item/ItemType.mapping

```diff
@@ -1,115 +1,122 @@
+	FIELD P hoeGold Lgm;
+	FIELD Q seeds Lgm;
+	FIELD R wheat Lgm;
+	FIELD S bread Lgm;
+	FIELD T helmetCloth Lgm;
+	FIELD U chestplateCloth Lgm;
+	FIELD V leggingsCloth Lgm;
+	FIELD W bootsCloth Lgm;
+	FIELD X helmetChain Lgm;
+	FIELD Y chestplateChain Lgm;
+	FIELD Z leggingsChain Lgm;
+	FIELD aA redstone Lgm;
+	FIELD aB snowball Lgm;
+	FIELD aC boat Lgm;
+	FIELD aD leather Lgm;
+	FIELD aE milk Lgm;
+	FIELD aF brick Lgm;
+	FIELD aG clay Lgm;
+	FIELD aH reeds Lgm;
+	FIELD aI paper Lgm;
+	FIELD aJ book Lgm;
+	FIELD aK slimeball Lgm;
+	FIELD aL minecartChest Lgm;
+	FIELD aM minecartFurnace Lgm;
+	FIELD aN egg Lgm;
+	FIELD aO compass Lgm;
+	FIELD aP fishingRod Lgm;
+	FIELD aQ clock Lgm;
+	FIELD aR yellowDust Lgm;
+	FIELD aS fishRaw Lgm;
+	FIELD aT fishCooked Lgm;
+	FIELD aU dyePowder Lgm;
+	FIELD aV bone Lgm;
+	FIELD aW sugar Lgm;
+	FIELD aX cake Lgm;
+	FIELD aY bed Lgm;
+	FIELD aZ diode Lgm;
+	FIELD aa bootsChain Lgm;
+	FIELD ab helmetIron Lgm;
+	FIELD ac chestplateIron Lgm;
+	FIELD ad leggingsIron Lgm;
+	FIELD ae bootsIron Lgm;
+	FIELD af helmetDiamond Lgm;
+	FIELD ag chestplateDiamond Lgm;
+	FIELD ah leggingsDiamond Lgm;
+	FIELD ai bootsDiamond Lgm;
+	FIELD aj helmetGold Lgm;
+	FIELD ak chestplateGold Lgm;
+	FIELD al leggingsGold Lgm;
+	FIELD am bootsGold Lgm;
+	FIELD an flint Lgm;
+	FIELD ao porkchopRaw Lgm;
+	FIELD ap porkchopCooked Lgm;
+	FIELD aq painting Lgm;
+	FIELD ar appleGold Lgm;
+	FIELD as sign Lgm;
+	FIELD at doorWood Lgm;
+	FIELD au bucket Lgm;
+	FIELD av bucketWater Lgm;
+	FIELD aw bucketLava Lgm;
+	FIELD ax minecart Lgm;
+	FIELD ay saddle Lgm;
+	FIELD az doorIron Lgm;
 	FIELD b rand Ljava/util/Random;
-	FIELD ba COOKIE Lgm;
-	FIELD bb MAP Lwr;
-	FIELD bc SHEARS Lbl;
-	FIELD bd RECORD_13 Lgm;
-	FIELD be RECORD_CAT Lgm;
+	FIELD ba cookie Lgm;
+	FIELD bb map Lwr;
+	FIELD bc shears Lbl;
+	FIELD bd record_13 Lgm;
+	FIELD be record_cat Lgm;
 	FIELD bf id I
 	FIELD bg maxStackSize I
-	FIELD c BY_ID [Lgm;
-	FIELD d IRON_SHOVEL Lgm;
-	FIELD e IRON_PICKAXE Lgm;
-	FIELD f IRON_HATCHET Lgm;
-	FIELD g FLINT_AND_STEEL Lgm;
-	FIELD h APPLE Lgm;
-	FIELD i BOW Lgm;
-	FIELD j ARROW Lgm;
-	FIELD k COAL Lgm;
-	FIELD l DIAMOND Lgm;
-	FIELD m IRON_INGOT Lgm;
-	FIELD n GOLD_INGOT Lgm;
-	FIELD o IRON_SWORD Lgm;
-	FIELD p WOODEN_SWORD Lgm;
-	FIELD q WOODEN_SHOVEL Lgm;
-	FIELD r WOODEN_PICKAXE Lgm;
-	FIELD s WOODEN_HATCHET Lgm;
-	FIELD t STONE_SWORD Lgm;
-	FIELD u STONE_SHOVEL Lgm;
-	FIELD v STONE_PICKAXE Lgm;
-	FIELD w STONE_HATCHET Lgm;
-	FIELD x DIAMOND_SWORD Lgm;
-	FIELD y DIAMOND_SHOVEL Lgm;
-	FIELD z DIAMOND_PICKAXE Lgm;
+	FIELD bk containerItemType Lgm;
+	FIELD c items [Lgm;
+	FIELD d shovelIron Lgm;
+	FIELD e pickaxeIron Lgm;
+	FIELD f hatchetIron Lgm;
+	FIELD g flintAndSteel Lgm;
+	FIELD h apple Lgm;
+	FIELD i bow Lgm;
+	FIELD j arrow Lgm;
+	FIELD k coal Lgm;
+	FIELD l diamond Lgm;
+	FIELD m ingotIron Lgm;
+	FIELD n ingotGold Lgm;
+	FIELD o swordIron Lgm;
+	FIELD p swordWood Lgm;
+	FIELD q shovelWood Lgm;
+	FIELD r pickxaeWood Lgm;
```

Comment:
!!!!!! @shedaniel uhh... hold my `pickxae` for a second
there's something off about this mapping